src/backend/core/tests/api/test_config.py

```diff
@@ -33,11 +33,12 @@ def test_api_config(is_authenticated):
 
     response = client.get("/api/v1.0/config/")
     assert response.status_code == HTTP_200_OK
-    assert response.json() == {
-        "ENVIRONMENT": "test",
-        "LANGUAGES": [["en-us", "English"], ["fr-fr", "French"], ["de-de", "German"]],
-        "LANGUAGE_CODE": "en-us",
-        "POSTHOG_KEY": "132456",
-        "POSTHOG_HOST": "https://test.i.posthog-test.com",
-        "POSTHOG_SURVEY_ID": "7890",
-    }
+    data = response.json()
+    assert data["ENVIRONMENT"] == "test"
+    assert data["LANGUAGES"] == [["en-us", "English"], ["fr-fr", "French"], ["de-de", "German"]]
+    assert data["LANGUAGE_CODE"] == "en-us"
+    assert data["POSTHOG_KEY"] == "132456"
+    assert data["POSTHOG_HOST"] == "https://test.i.posthog-test.com"
+    assert data["POSTHOG_SURVEY_ID"] == "7890"
+    assert type(data["AI_ENABLED"]) is bool
+    assert type(data["AI_FEATURE_SUMMARY_ENABLED"]) is bool
```

Comment:
you should leave it like it was, it's a good safeguard against adding unexpected configs in the public scope